src/RelayServer.ts

```diff
@@ -972,3 +993,21 @@ latestBlock timestamp   | ${latestBlock.timestamp}
         this.ready = isReady;
     }
 }
+
+/**
+ * TODO: Hard-coded values: for testing purposes only!
+ */
+function getWeiFromRifWei(trifWei: BN): BN {
+    const tRifPriceInRBTC = 0.000005739;
+    const rifTokenDecimals = 18;
+
+    const costInTrif = parseFloat(fromWei(trifWei));
+    const costInRBTC = costInTrif * tRifPriceInRBTC;
+    const costInRBTCFixed = costInRBTC.toFixed(rifTokenDecimals);
+    const costInWei = toWei(costInRBTCFixed);
+    return toBN(costInWei);
+}
+
+function getGas(cost: BN, gasPrice: BN) {
+    return cost.div(gasPrice);
+}
```

Comment:
as with the client PR, i'd extract this to a separate file even if it is just temporary

Reply:
Done, thanks for the suggestion